source/iterators.tex

```diff
@@ -2081,7 +2081,7 @@
 The only valid use of an \tcode{operator*}
 is on the left side of the assignment statement.
 Assignment through the same value of the iterator happens only once.
-Equality and inequality might not be defined.
+It is possible that equality and inequality are not defined.
```

Comment:
I find this a bit clunky.  I think it would read better as "Equality and inequality are not necessarily defined."

Reply:
fixed + force-pushed